docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -1,35 +1,28 @@
 [[detections-ui-exceptions]]
 == Rule exceptions
 
-Rule exceptions can be associated with detection and endpoint rules to prevent those rules from generating alerts under specific circumstances. Exceptions are made up of three main components: exception items, default rule lists, and shared exception lists.
+Rule exceptions can be associated with detection and endpoint rules to prevent those rules from generating alerts under specific circumstances. You can use exceptions to reduce the number of false positives, and to prevent trusted processes and network activity from generating unnecessary alerts.
+
+When creating exceptions, you can assign them to <<rule-exceptions-intro,individual rules>> or share them among <<shared-exception-list-intro,multiple rules>>.
 
 [float]
-[[exception-item-intro]]
-=== Exception items
+[[rule-exceptions-intro]]
+=== Exceptions for individual rules
 
-Exception items, also referred to as _exceptions_, contain the source event conditions that determine when alerts are not generated. You can use exceptions to reduce the number of false positives, and to prevent trusted processes and network activity from generating unnecessary alerts. 
+Exceptions, also referred to as _exception items_, contain the source event conditions that determine when alerts should or should not be generated. 
 
-Rules can have multiple exceptions and exceptions can apply to multiple rules. Refer to <<add-exceptions>> to learn more about adding exceptions to rules. 
+You can create exceptions that apply exclusively to a single rule. These types of exceptions cannot be used by other rules and you must manage them from the rule’s details page. To learn more about creating and managing these types of exceptions, refer to <<add-exceptions>>.
```

Comment:
We're hoping to be able to allow a user to manage them from the exceptions lists management page, but we're not there yet.

Reply:
Makes sense! Just to double-check, is there anything in this section that implies that users should go to the shared exception list page for managing exceptions? The reference in the last sentence points to [these docs](https://www.elastic.co/guide/en/security/master/add-exceptions.html), which explain how to manage individual rule exceptions.

Reply:
Nothing there implies managing them in the shared exception list page. I think it just made me recall that we hope to have that capability in the near future.

Reply:
Ah ok, cool. Thanks for clearing that up! I'll make a mental note of this and resolve this convo. Thanks!